engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -93,11 +91,13 @@ private void updateFullScale() {
      */
     public void update() {
         updateFullScale();
-        if (READ_ONLY_GBUFFER.getFbo().dimensions().areDifferentFrom(fullScale)) {
+        if (get(new ResourceUrn("engine:sceneOpaque")).dimensions().areDifferentFrom(fullScale)) {
             disposeAllFBOs();
             createFBOs();
-            updateDefaultFBOs();
             notifySubscribers();
+
+            WorldRendererImpl worldRendererImpl = ((WorldRendererImpl)context.get(WorldRenderer.class));
+            worldRendererImpl.initRenderGraph();
```

Comment:
No, sorry, I'm against this. You are refreshing the whole render graph whenever the display changes size, when all that needs refreshing are the IDs stored in the FBOs and in the state changes/tasks that use them.

This is not a fix, it's hiding the dirt under the carpet. :smiling_imp: